docs/pre-release/releaseNotes.yml

```diff
@@ -30,6 +30,35 @@ docDescription: >-
 changelog: https://github.com/telepresenceio/telepresence/blob/$branch$/CHANGELOG.md
 
 items:
+  - version: 2.3.4
+    date: '2021-07-09'
+    notes:
+      - type: bugfix
+        title: Improved IP log statements
+        body: >-
+          Fixed log statements that were printing incorrect IP addresses.
+      - type: bugfix
+        title: Improved messaging when multiple services match a workload
+        body: >-
+          If multiple services matched a workload when performing an intercept, Telepresence would panic.
+          It now gives the correct error message, instructing the user on how to specify which
+          service the intercept should use.
+      - type: bugfix
+        title: Traffic-manger determine's service subnet in own namespace
+        body: >-
+          Telepresence will now determine the service subnet by creating a dummy-service in its own
+          namespace, instead of the default namespace, which was causing RBAC permissions issues in
+          some clusters.
+      - type: bugfix
+        title: Telepresence connect respects pre-existing clusterrole
+        body: >-
```

Comment:
I see that we've made updates to the RBAC page in this PR; should we link to that page here or perhaps another one?

Reply:
I can link to the RBAC changes here since they are mentioned, but the RBAC changes in this PR were more associated with the helm chart fixes (but I still think linking to the helm chart there is better)